bundles/binding/org.openhab.binding.isy/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/isy_inst.jar
```

Comment:
Support Request [#16367] created with Universal Devices regarding this question.